### ISSUE TITLE
Controller Suport Request "Moga XP5-X Plus"

### ISSUE THREAD

**Issue by Conan179** [ndeadly/MissionControl#50]
Hello@all.

Would it be possible to add support for the Moga XP5-X Plus?
https://www.xbox.com/de-DE/accessories/mobile-gaming/moga-xp5-x

**Comment by ndeadly** [comment]
Can you dump your pairing database info with btdb.nro and post it here?

**Comment by Conan179** [comment]
I cand pair it.
![2020090413422900-1E95E5926F1CB99A87326D927F27B47E](https://user-images.githubusercontent.com/1906242/92235883-b4056d80-eeb4-11ea-8d82-6fddab589570.jpg)
BT paring is working on android Smartphone and windows 10 PC (no adapters)

**Comment by ndeadly** [comment]
The controller won't show up as paired until it has been whitelisted by me. It may still show up in the pairing database though if it manages to complete the pairing process.

However, if it's Bluetooth LE only then it won't work at the moment, as I am yet to figure out how to make LE connections. This is the same issue holding me back from adding support for controllers like the Steam Controller, and Bluetooth keyboards/mice. Hopefully it is possible in the future once I understand how the LE stuff works on the Switch.

**Comment by Conan179** [comment]
![2020090414230400-6F1D89FC03AFE4BD4D8C0BB78FEF3260](https://user-images.githubusercontent.com/1906242/92239675-b3240a00-eebb-11ea-94c1-ac705a197ba2.jpg)

ok.

**Comment by ndeadly** [comment]
Yeah, looks like it's probably trying to connect via Bluetooth LE then. I'll get back to this once I've figured out how to make LE work.

**Comment by Conan179** [comment]
Ok, Thanks for helping :)

**Comment by ndeadly** [comment]
Hey, making some progress with BLE. Would it be possible for you to install [this mobile app](https://play.google.com/store/apps/details?id=no.nordicsemi.android.mcp&hl=en) on your phone to connect and read the device characteristics and make me a debug log for this controller?

**Comment by Conan179** [comment]
No Problem. I hope i male it correct
![Screenshot_20201201-194227_nRF Connect](https://user-images.githubusercontent.com/1906242/100783712-6abeae80-340e-11eb-90d9-ce816a704716.jpg)
[Log 2020-12-01 19_43_54.txt](https://github.com/ndeadly/MissionControl/files/5624731/Log.2020-12-01.19_43_54.txt)

**Comment by ndeadly** [comment]
That's perfect, thanks! Just gathering information on a few BLE controllers at the moment to get a feel for the services they expose. I suspect my next couple of days will be spent working on 11.0.0 support, but maybe by the weekend I might have something for you to test.
